### PR TITLE
163279:Support new Application Gateway Tiers and options

### PR DESCRIPTION
**Pull request by ashutosh-kumar1**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Added support for Standard_v2 and WAF_v2 application gateway tiers
* Added options for the above-mentioned tiers

### CODE REVIEW

File: lansa-vmss-windows-autoscale-sql-database/createUiDefinition.json [hunk 1]
```diff
@@ -593,7 +622,66 @@
...
+                            "required": false,
+                            "regex": "^(?!\\s*$).+",
+                            "validationMessage": "The value must not be empty."
+                        },
+                        "visible": "[equals(steps('applicationGatewayConfig').autoScalingEnabled, true)]"
+                    },
+                    {
+                        "name": "zones",
+                        "type": "Microsoft.Common.DropDown",
+                        "label": "Availability Zone",
```
Comment (robe070) [inline lansa-vmss-windows-autoscale-sql-database/createUiDefinition.json]: Needs to be plural - `Zones`

**Review by robe070 — COMMENTED** [review]
Just fix the plural. We can then get it published and see what it does in the real environment

File: lansa-vmss-windows-autoscale-sql-database/createUiDefinition.json (hunk 1 above)
Comment (ashutosh-kumar1) [reply lansa-vmss-windows-autoscale-sql-database/createUiDefinition.json]: Label updated.

**Comment by ashutosh-kumar1** [comment]
@robe070, Commit done to fix plural.

**Comment by ashutosh-kumar1** [comment]
@robe070 , Please merge the PR. The checks have passed.

**Pull request merged**